dev/TreeView/TreeViewAutomationPeers.idl

```diff
@@ -19,4 +19,11 @@ unsealed runtimeclass TreeViewItemAutomationPeer : Windows.UI.Xaml.Automation.Pe
     [method_name("CreateInstanceWithOwner")] TreeViewItemAutomationPeer(MU_XC_NAMESPACE.TreeViewItem owner);
 }
 
-}
\ No newline at end of file
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+unsealed runtimeclass TreeViewItemDataAutomationPeer : Windows.UI.Xaml.Automation.Peers.ItemAutomationPeer, Windows.UI.Xaml.Automation.Provider.IExpandCollapseProvider
```

Comment:
@jevansaks  Can I make change the runtime class base:
unsealed runtimeclass TreeViewListAutomationPeer : Windows.UI.Xaml.Automation.Peers.SelectorAutomationPeer
to
unsealed runtimeclass TreeViewListAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewBaseAutomationPeer?

Here TreeViewItemDataAutomationPeer's base class is ItemAutomationPeer, and maybe ListViewItemDataAutomationPeer is more reasonable. To support that, I need to change the base class TreeViewListAutomationPeer too. Otherwise I can't pass the compilation.